corehq/apps/app_manager/models.py

```diff
@@ -2713,12 +2713,27 @@ def validate_for_build(self):
         return errors
 
 
+class ReportGraphConfig(DocumentSchema):
```

Comment:
I pulled out the properties needed for this feature into ```ReportGraphConfig```.  Other properties (like all except ```config``` in ```GraphSeries```) are not required for this feature.